File: apps/api/src/routes/health.ts
import type { FastifyInstance } from "fastify";

export async function healthRoutes(app: FastifyInstance): Promise<void> {
  app.get("/health", async (_request, reply) => {
    return reply.send({
      status: "ok",
      service: "flowiq-api",
      timestamp: new Date().toISOString(),
      version: process.env.npm_package_version ?? "0.0.1",
    });
  });
}

File: apps/ai-service/app/routers/health.py
from datetime import datetime, timezone

from fastapi import APIRouter

router = APIRouter()


@router.get("/health")
async def health_check() -> dict:
    return {
        "status": "ok",
        "service": "flowiq-ai-service",
        "timestamp": datetime.now(timezone.utc).isoformat(),
        "version": "0.0.1",
    }
